src/libtess2/tess.c

```diff
@@ -482,6 +482,7 @@ TESStesselator* tessNewTess( TESSalloc* alloc )
 	if ( tess == NULL ) {
 		return 0;          /* out of memory */
 	}
+	tess->outOfMemory = 0;
```

Comment:
Since there was already one other edit to this lib this seemed like the cleanest way to make sure this value was initialized.